### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TinySadPotatoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1383,4 +1383,7 @@ here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-p
 
 I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join community! :smiling_face_with_three_hearts:
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
----
\ No newline at end of file
+---
+#Hi, Kottans!
+I'm Kyrylo, I have always dreamed of programming and this is a great opportunity for me :)
+<img src="https://f3.upet.com/o_XSbhGBwd1G_1.jpg" width="75">
```
Comment (Temu4) [inline README.md:1389]: Hi Kyrylo!
Really good job! 👍 
More preferable is using markdown syntax.  Can you please refactor attached image in something like [this](https://www.markdownguide.org/basic-syntax/#images)
Thanks!

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TinySadPotatoe) [reply README.md:1389]: HI!
Thanks for the help, I'll fix it now

Comment (Temu4) [reply README.md]: 👍

**Comment by Temu4** [comment]
Can you also pull the latest changes from original repo and resolve merge conflicts?

**Comment by TinySadPotatoe** [comment]
I hope, I did it

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
One more step. You have duplicates in your changes. Probably these are the consequences of an incorrectly merge.

**Comment by TinySadPotatoe** [comment]
I saw that text was duplicated before the merge. I thought that there was no need to change what I did not write. Now I think it's all fixed

File: README.md [hunk 2]
```diff
@@ -1390,6 +1390,13 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
 ---
 
-## What's up Kottans!
-My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+
+## What's up Kottans!
```
Comment (nazmariam) [inline README.md:1394]: Almost, but still affecting old changes. Rows from 1393 to 1395 shouldn't change

**Comment by TinySadPotatoe** [comment]
I have moved my post to these lines, I hope this corrects the situation

**Review by Temu4 — APPROVED** [review]
Looks good for me now! 👍

File: README.md [hunk 3]
```diff
@@ -1390,6 +1390,12 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
...
-## What's up Kottans!
-My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+## Hi, Kottans!
+I'm Kyrylo, I have always dreamed of programming and this is a great opportunity for me :)
+ ![Cat in the glasses](https://f3.upet.com/o_XSbhGBwd1G_1.jpg)
+ 
+## What's up Kottans! 
+My name is [Alex](https://github.com/usides) and I want to be a dev-cat.
```
Comment (nazmariam) [inline README.md:1398]: Still rewriting old changes, please remove indents from 1398 to 1401 at the beginnings of the lines

Comment (nazmariam) [reply README.md:1398]: Look, you shouldn't change previously added code
Screenshot 2020-10-13 at 16 36 10
Now GitHub shows us that you've removed Alex's changes and added them after yours

Comment (TinySadPotatoe) [reply README.md:1398]: Above they wrote that Alex's code was duplicated and asked me to delete, which I did. Maybe I did something wrong. Maybe I delete this request and submit a new one?
https://github.com/kottans/mock-repo/pull/442#pullrequestreview-507354098
I think that's why the error occurred

Comment (TinySadPotatoe) [reply README.md:1398]: I'm rewrite this, but on my side I don't see indents. Sorry for that, it's my first pull request 😿

Comment (TinySadPotatoe) [reply README.md:1398]: It would be great
t.me/Sadpotatoe

**Review by nazmariam — APPROVED** [review]
Perfect!

**Review by miroslav-kolomiets — APPROVED** [review]
Nice job!

**Pull request merged**